### PR TITLE
new recipe

### PR DESCRIPTION
**Pull request by vahtras**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/daltools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f3f21cfae3af36b4a941573470dd70a8910307204bbc0a3568d7db7c8ed7f8f9
+
+build:
+  number: 1
```
Comment (chrisburr) [inline recipes/daltools/meta.yaml:13]: This should start from 0
```suggestion
  number: 0
```

File: recipes/daltools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3
```
Comment (chrisburr) [inline recipes/daltools/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/daltools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    - pip
+  run:
+    - numpy
+    - python >=3
```
Comment (chrisburr) [inline recipes/daltools/meta.yaml:23]:
```suggestion
    - python >=3.6
```

**Comment by peterjc** [comment]
Also, as per the first item on the check list, could you change the PR title from "new recipe" to something like "adding daltools - Utilities for processing Dalton files" please?

**Comment by vahtras** [comment]
Closing this due to duplicate PR:s #13608

**Pull request closed without merging**